docs/usage.md

```diff
@@ -64,12 +64,35 @@ Options:
   --help                          Show this message and exit.
 ```
 
+
+## `watchmaker` as a standalone package
+
+Once a Watchmaker standalone executable has been
+[downloaded](installation.html) and a
+[configuration file](configuration.html) has been created (or you have decided
+to use the default configuration), using Watchmaker is the same as
```

Comment:
this bit seems to be missing something: `using Watchmaker is the same as using Watchmaker as a CLI utility`...